packages/realm/bindgen/src/templates/node.ts

```diff
@@ -31,6 +31,7 @@ import {
   Pointer,
   Template,
 } from "@realm/bindgen/bound-model";
+import { clangFormatter } from "@realm/bindgen/formatter";
```

Comment:
It feels a bit weird to call this `clangFormatter` since the tool is named `clang-format`, and the tool called `clang` doesn't do any formatting. I admit that `clangFormatFormatter` sounds ridiculous though. Maybe we should just call the formatters by the name of the tool with no suffix, so just `clangFormat` and `eslint`?